docs/sources/features/datasources/cloudwatch.md

```diff
@@ -114,20 +109,62 @@ aws_secret_access_key = dasdasdsadasdasdasdsa
 region = us-west-2
 ```
 
-## Metric Query Editor
+## Using the M  etric Query Editor
```

Comment:
It looks ok when I build locally but why is there a space between M and etric here?